src/SwashBuckle.AspNetCore.MicrosoftExtensions/Extensions/MetadataAttributeExtensions.cs

```diff
@@ -14,7 +14,19 @@ internal static class MetadataAttributeExtensions
             if (attribute.Summary != null)
                 yield return new KeyValuePair<string, object>(Constants.XMsSummary, attribute.Summary);
             if (attribute.Description != null)
-                yield return new KeyValuePair<string, object>(Constants.XMsDescription, attribute.Description);
+                yield return new KeyValuePair<string, object>(Constants.Description, attribute.Description);
+        }
+
+        internal static IEnumerable<KeyValuePair<string, object>> GetSwaggerOperationExtensions (this MetadataAttribute attribute)

```

Comment:
Specify in comment why operation description differs from other item descriptions and throw in a link to documentation

Reply:
Only the summary summary differs
"summary" for operations
"x-ms-summary" for parameters
https://flow.microsoft.com/en-us/documentation/customapi-how-to-swagger/